pkg/handler/repositories.go

```diff
@@ -0,0 +1,90 @@
+package handler
+
+import (
+	"github.com/content-services/content-sources-backend/pkg/db"
+	"github.com/content-services/content-sources-backend/pkg/models"
+	"github.com/labstack/echo/v4"
+)
+
+type RepositoryItem struct {
+	UUID                string `json:"uuid"`
+	Name                string `json:"name"`
```

Comment:
What's the difference between a RepositoryItem and a RepositoryConfiguration? And is this struct meant to only work with RepositoryConfigurations, or will there be others?

Reply:
generally from what i've seen, you separate your api return data from your database models.  So the RepositoryConfiguration is the database representation, the RepositoryItem is the api representation.  And you have methods that can convert from one to the other.  This lets you make changes to the data model without affecting the UI, as well as combining multiple tables worth of data into a single 'item' in the repsonse (which will be used when we include information about # of packages, etc..)